webview/src/views/history.svelte

```diff
@@ -228,22 +227,59 @@
   }
 </script>
 
-<title>Project Workflow panel</title>
-<h1>{title(history)}</h1>
-<h2>duration: {duration(history)}</h2>
-{#each workflowTasks as workflowTask}
-  <div class:current={workflowTask.startedEventId === currentWorkflowTaskStartedEventId}>
-    <BreakpointButton {workflowTask} />Workflow Task ({workflowTask.status})
-    {#each workflowTask.events as event}
-      <ul>
-        {labelTextForHistoryEvent(event)}
+<section>
+  {#if history}
```

Comment:
Just don't render this component with no history, there's no point.

Reply:
It's [not actually being loaded](https://github.com/temporalio/vscode-debugger-plugin/pull/18/files#diff-2e51702f237c2d3514d1405122e069f7d202d58a0ce5f96458f29bf5c671c164R45). I left this in for testing the UI, will remove 🗑️  

Although I think we probably want a fallback UI here in case there is no `history` for some reason.

Reply:
I guess.. for now I think it's good enough to assume this component is only rendered when there's `history` to show but it's not critical.